### PR TITLE
CH-1042: For completed campaigns, render the winner in PHP where possibl...

### PR DESCRIPTION
**Pull request by katbailey**
...e

### CODE REVIEW

File: personalize.module [hunk 1]
```diff
@@ -1688,7 +1701,7 @@ function personalize_get_agent_types() {
 /**
  * Load a particular agent from the database or from code.
```
Comment (eshtadc) [inline personalize.module:1702]: I always figured that if you specifying a single object (campaign in this case) that you always want it regardless of status.  So from that - I would say that anytime you load a specific campaign by name you want it regardless of whether it is completed or not.  Now - that may not play with the way our code works... might be something weird somewhere...

File: modules/personalize_blocks/test/personalize_blocks.test [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-class PersonalizeBlocksTest extends DrupalWebTestCase {
+class PersonalizeBlocksTest extends PersonalizeBaseTest {
```
Comment (eshtadc) [inline modules/personalize_blocks/test/personalize_blocks.test:3]: brilliant.

File: personalize.module (hunk 1 above)
Comment (katbailey) [inline personalize.module:1702]: So, we shouldn't even have this $include_completed parameter here? That feels more correct to me too. Will see if it breaks anything...

Comment (katbailey) [inline personalize.module:1702]: I've made that change - there was no code that seemed to care about that parameter when loading a single agent, which makes sense - seeing as we've never even had completed agents.

**Pull request closed without merging**